index.html

```diff
@@ -63,5 +63,175 @@ <h6><a href="#">LET’S CONNECT</a></h6>
+              </ul>
+            </div>
+          </div>
+
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+              <li class="tag-1"><h6>html</h6></li>
+              <li class="tag-2"><h6>css</h6></li>
+              <li class="tag-3"><h6>javaScript</h6></li>
+            </ul>
+            <div class="action">
+              <button type="button" class="button-text">See Project</button>
+            </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="about">
+    <div class="about-content">
+      <div class="about-top">
+        <h2>About Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need
+           coded, don’t hestiate to contact me.
+        </p>
+        <div class="social-media">
+          <div class="let-talk-text">
+            <h6><a href="#">LET’S CONNECT</a></h6>
+
+            <ul>
+              <li> <a href="#"><img src="images/mobile/twitter.png" alt="image"/></a></li>
+              <li> <a href="#"><img src="images/mobile/linkedin.png" alt="image"/></a></li>
+              <li> <a href="#"><img src="images/mobile/github.png" alt="image"/></a></li>
+              <li> <a href="#"><img src="images/mobile/m.png" alt="image"/></a></li>
+              <li> <a href="#"></a> <img src="images/mobile/fingers.png" alt="image"/></li>
+           </ul>
+          </div>   
+        </div>
+        <div class="action">
+          <button type="button" class="button-text">Get my resume</button>
+        </div>
+      </div>
+      <div class="frame-12">
+        <div class="frame-7">
+          <h6>Langguages</h6>
+          <button class="frame-7-button" type="button"><i class="fas fa-caret-down"></i></button>
+        </div>
+        <div class="list-language">
+          <ul>
+            <li class="list-language-1"><img src="images/mobile/work/js.png" alt="image"/>
+              JavaSript
+            </li>
+            <li class="list-language-2">
+              <img src="images/mobile/work/html.png" alt="image"/>
+              HTML
+            </li>
+            <li class="list-language-3">
+              <img src="images/mobile/work/css.png" alt="image"/>
+              CSS
+            </li>
+          </ul>
+        </div>
+
+        <ul class="frame-8">
+          <li>Frameworks
+            <i class="fas fa-arrow-right"></i>
+            <img src="images/mobile/work/separator-bottom.png" class="separator-1" alt="images"/>
+          </li>
+        </ul>
+
+        <ul class="frame-9">
+          <li>
+            Skills
+            <i class="fas fa-arrow-right"></i>
+            <img src="images/mobile/work/separator-bottom.png" class="separator-2" alt="images"/>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
```

Comment:
You have not implemented some of the requested changes from [the previous review](https://github.com/Emmy-github-webdev/portfolio/pull/2#discussion_r711417213)

**This is your ABOUT MYSELF section**:

![displaced-about](https://user-images.githubusercontent.com/36542986/133886988-8cd8c880-9818-404e-a910-66143595ae42.jpg)

- [ ] It does not align with the WORKS section. Please fix this.
- [ ] Your 'LET'S CONNECT' text should be bigger and aligned to the left as suggested
![lets-connect](https://user-images.githubusercontent.com/36542986/133887115-5d44c2b3-2ddb-4ff8-b55d-73cc790b8a90.jpg)

#### Look at the image below:
- [ ] Your JavaScript, HTML and CSS tags are aligned to the right. Please, align them to the left as in the template
- [ ] You aligned your Frameworks and Skills parts to the left just like the Languages. as suggested but your Language text is now out of line. Please fix it.
- [ ] Please use the correct arrow icons as suggested. You can get these icons using the link from [Boxicons](https://boxicons.com/?query=chevron) as provided.

This is your image:
![jfs](https://user-images.githubusercontent.com/36542986/133887284-ebc2c4bf-9a57-4297-bd7c-566360bbeef5.jpg)

Reply:
- [ ] As suggested by the previous reviewer, as per the design, and as a requirement, please make sure the languages are aligned to the left. 
- [ ] The test on "Get my resume" button is not centralized, please kindly fix that.
**Your Implementation**
![suni1](https://user-images.githubusercontent.com/65852621/133917060-2d121cd1-15a1-42ef-837f-09a490ef8a45.png)
**The design**
![sund1](https://user-images.githubusercontent.com/65852621/133917040-3af77164-4102-460d-b189-13ae105f8b57.png)